packages/shared-ui/src/components/canvas/components/FlowCanvas.tsx

```
@@ -69,8 +69,8 @@ import TidyIcon from '../../../assets/icons/TidyIcon';
 
 import { INodeType } from '../types';
 import { useFlowProject } from '../context/FlowProjectContext';
-import { usePrefs } from '../../../contexts/PrefsContext';
-import { isInVSCode } from '../../../themes/vscode';
+import { usePrefs } from 'shell';
```

ESLint

[error] 'shell' import is restricted from being used. Renamed: import from 'shell'.

(no-restricted-imports)